.github/workflows/tests.yml

```diff
@@ -201,6 +201,16 @@ jobs:
       - name: Install Bundler RubyGems
         run: brew install-bundler-gems
 
+      - name: Create parallel test log directory
+        run: mkdir tests
+
+      - name: Cache parallel tests log
+        uses: actions/cache@v1
+        with:
+          path: tests/parallel_runtime_rspec.log
+          key: ${{ runner.os }}-parallel_runtime_rspec.log
```

Comment:
Just noticed: probably want to include the `test-flags` in the key somehow (because otherwise the three matrix workers will overwrite each other's cache)